src/mixins/document-mixin.ts

```diff
@@ -1,11 +1,8 @@
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
+import { Component, Vue } from 'vue-property-decorator'
 import { AxiosResponse } from 'axios'
 import { AxiosInstance as axios } from '@/utils'
 import { DocumentUpload, PdfInfoIF } from '@/interfaces'
 import { PdfPageSize } from '@/enums'
-import pdfjsLib from 'pdfjs-dist/build/pdf'
-pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/build/pdf.worker.entry')
```

Comment:
With a `require` up here, Jest allocates the memory and does not free it between test suites.

So, I moved the requires into the class `created()` lifecycle hook. As far as I can tell, there's no need to free the memory explicitly when the class is unmounted/destroyed as that's done automatically.